deploifai/cli/application/create.py

```diff
@@ -0,0 +1,316 @@
+import os
+import click
+import re
+import docker
+from time import sleep
+from InquirerPy import prompt
+from click_spinner import spinner
+
+from deploifai.cli.api import DeploifaiAPIError
+from deploifai.cli.context import pass_deploifai_context_obj, DeploifaiContextObj, is_authenticated, project_found, local_config
+
+
+@click.command()
+@click.argument("name")
+@click.option(
+    "--env",
+    help="Environment variables for the container (e.g. --env path/to/env OR --env VAR1=VAL1 --env VAR2=VAL2)",
+    multiple=True,
+)
```

Comment:
Add another `--env-file` option to read env files. This should be able to accept multiple files as well.

If the user uses both `--env` and `--env-file` options, just read everything and aggregate into one list.